gems/actionpack/6.0/actionpack-generated.rbs

```diff
@@ -511,18 +511,6 @@ module AbstractController
     # sticks the result in <tt>self.response_body</tt>.
     def render: (*untyped args) { () -> untyped } -> untyped
 
-    # Raw rendering of a template to a string.
-    #
-    # It is similar to render, except that it does not
-    # set the +response_body+ and it should be guaranteed
-    # to always return a string.
-    #
-    # If a component extends the semantics of +response_body+
-    # (as ActionController extends it to be anything that
-    # responds to the method each), this method needs to be
-    # overridden in order to still return a string.
-    def render_to_string: (*untyped args) { () -> untyped } -> untyped
```

Comment:
It seems this method takes a block, but it never uses it. So it's okay to make it optional.
https://github.com/rails/rails/blob/v8.0.2/actionpack/lib/abstract_controller/rendering.rb#L44